diagram.md

```diff
@@ -208,11 +208,11 @@ Below is a representation of the Block structure.
 * Data
   * Frame
 
-Although the Timecode value is stored once per Cluster, another timecode is stored within the Block structure itself. The way this works is that the Timecode in the Cluster is relative to the entire `Segment`. It is usually the Timecode that the first Block in the Cluster needs to be played at. The Timecode in the Block itself is relative to the Timecode in the Cluster. For example, let's say that the Timecode in the Cluster is set to 10 seconds, and you have a Block in that Cluster that is supposed to be played 12 seconds into the clip; this means that the Timecode in the Block would be set to 2 seconds.
+The `Timecode Element` value MUST be stored once per `Cluster`. Another `Timecode` SHOULD be stored within the `Block` structure itself. The `Timecode` in the `Cluster` is relative to the entire `Segment`. The `Timecode` SHOULD be the first `Block` in the `Cluster`. The `Timecode` in the `Block` itself is relative to the `Timecode` in the `Cluster`. For example, if the `Timecode` in the `Cluster` is set to 10 seconds and a `Block` in that `Cluster` is supposed to be played 12 seconds into the clip, the `Timecode` in the `Block` would be set to 2 seconds.
```

Comment:
> The `Timecode Element` SHOULD be the first `Element` in the `Cluster`.

In order_guidelines.md, it says the Timecode element MUST be the first Element in the Cluster.